src/Model/RateLimit.php

```diff
@@ -8,8 +8,8 @@ class RateLimit
 {
     /** @var array<string> */
     private array $vary = [];
-    private int $limit;
-    private int $period;
+    private readonly int $limit;
+    private readonly int $period;
 
     public function __construct(int $limit, int $period)
     {
```

Comment:
This could probably be inlined (rector just didn't handle the if condition)
```suggestion
    public function __construct(private readonly int $limit, private readonly int $period)
    {
```

Reply:
done